src/components/Dropdown.svelte

```diff
@@ -8,16 +8,14 @@
 </script>
 
 <script lang="ts" strictEvents>
-  import type { State } from "@app/lib/cobs";
-
   import { createEventDispatcher } from "svelte";
   import { twemoji } from "@app/lib/utils";
   import Badge from "@app/components/Badge.svelte";
 
-  type T = $$Generic<State | string | number>;
+  type T = $$Generic;
```

Comment:
:exploding_head:

Reply:
lol 🚬 🤡

Reply:
Nothing better than to be explicit right? heeh